packages/@lwc/shared/scripts/rollup/rollup.config.js

```diff
@@ -26,7 +26,13 @@ function rollupConfig({ format }) {
             banner,
             footer,
         },
-        plugins: [typescript({ target: 'es2017', typescript: require('typescript') })],
+        plugins: [
+            typescript({
+                target: 'es2017',
```

Comment:
I could do this, but then we would have to change the server-targeted modules (like `@lwc/compiler`) to target es2018  instead, since that's what they're inheriting from the root tsconfig. I think it might be less confusing if the root tsconfig didn't define any target at all.